lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb

```diff
@@ -6,4 +6,5 @@
   "british-or-irish": "a British or Irish citizen",
   "eea": "from another EU country, the EEA or Switzerland",
   "non-eea": "a non-EEA family member of someone from the EU, EEA or Switzerland",
+  "somewhere-else": "From somewhere else or you don’t knows",
```

Comment:
This text is included in the first commit as well, if you'd like to amend it there also.